src/build_methods/self.rb

```diff
@@ -0,0 +1,23 @@
+
+# frozen_string_literal: true
+
+require 'system_command'
+
+module Metalware
+  module BuildMethods
+    class Self < BuildMethod
+      TEMPLATES = [:self].freeze
+
+      def render_build_started_templates(parameters)
+        render_template(:self, parameters: parameters)
+      end
+
+      def start_build
+        rendered_self_template = file_path.template_save_path(:self, node: node)
+        puts SystemCommand.run("bash #{rendered_self_template}")
```

Comment:
Rather than running the rendered script through Bash, can we just run the script directly and obey its `#!` line? This currently works fine as our single repo uses a Bash script, but this would allow a repo to e.g. specify a script with `#! /usr/bin/env ruby` and have it run as expected.

I believe the following should be sufficient to do this, but you may want to quickly test this:

```ruby
rendered_self_template = file_path.template_save_path(:self, node: node)
FileUtils.chmod '+x', rendered_self_template
puts SystemCommand.run(rendered_self_template)

```

Reply:
Fixed in: https://github.com/alces-software/metalware/pull/181/commits/722d944c66cd0f3a6dc3041476bf69ce63950be6